### ISSUE TITLE
Add mode method

### ISSUE THREAD

**Issue by aloctavodia** [arviz-devs/preliz#604]
In #603  we introduced a mode argument to maxent and a mode method to many distributions. The mode is still missing from the following distributions.

* [x] AsymmetricLaplace
* [x] BetaScaled
* [x] ChiSquared
* [x] ExGaussian
* [x] Kumaraswamy
* [x] LogLogistic
* [x] LogitNormal
* [x] Pareto
* [x] Rice
* [x] SkewNormal
* [x] SkewStudentT
* [x] StudentT
* [x] TruncatedNormal
* [x] Uniform
* [x] VonMises
* [x] Wald
* [x] Weibull
* [x] BetaBinomial
* [x] Binomial
* [x] Categorical
* [x] DiscreteUniform
* [x] DiscreteWeibull
* [x] HyperGeometric
* [x] ZeroInflatedBinomial
* [x] ZeroInflatedNegativeBinomial
* [x] ZeroInflatedPoisson
* [x] Dirichlet
* [x] MultiVariateNormal
* [ ] Hurdle
* [ ] Censored
* [x] Truncated
* [ ] Mixture

**Comment by rohanbabbar04** [comment]
I have a question: there are some distributions that are bimodal. Should we return both modes or just one?

**Comment by aloctavodia** [comment]
Good question. Maybe only the major mode? Not sure what to do if two equal modes. 

Also for the beta and beta scaled, I think we are currently returning 0.5 when alpha and beta are lower than 1. But we should do:

0.5  if α=β = 1
No mode if α<1 OR β<1.
at 0 for α ≤ 1, and at 1 if β ≤ 1

**Comment by aloctavodia** [comment]
yep, probably the better thing is to respect multimodality. Then we will need to check that setting the mode works for `maxent` or that at least a warning is raised if we can not handle those cases.

**Comment by Advaitgaur004** [comment]
> One option is to go with the major one, other is to return a tuple of both modes

I believe returning a list or tuple of modes is beneficial. For a bimodal normal mixture, both peaks should be returned. That’s my suggestion.

**Comment by Advaitgaur004** [comment]
> yep, probably the better thing is to respect multimodality. Then we will need to check that setting the mode works for `maxent` or that at least a warning is raised if we can not handle those cases.

I have now implemented unimodal modes, but for bimodal, we need to return tuples.
